ansible/templates/whisk.properties.j2

```diff
@@ -18,6 +18,7 @@ defaultLimits.actions.invokes.perMinute={{ defaultLimits.actions.invokes.perMinu
 defaultLimits.actions.invokes.concurrent={{ defaultLimits.actions.invokes.concurrent }}
 defaultLimits.triggers.fires.perMinute={{ defaultLimits.triggers.fires.perMinute }}
 defaultLimits.actions.invokes.concurrentInSystem={{ defaultLimits.actions.invokes.concurrentInSystem }}
+limits.actions.sequence.maxLength={{ defaultLimits.actions.sequence.maxLength }}
```

Comment:
if this is intended as a default limit (following the other properties), should be defined the same way (with a default and an env specific limit)?